libportability-gfx/src/impls.rs

```diff
@@ -1509,39 +1532,14 @@ pub extern "C" fn gfxCreateGraphicsPipelines(
                     VK_SHADER_STAGE_TESSELLATION_CONTROL_BIT => { set.hull = Some(entry_point); }
                     VK_SHADER_STAGE_TESSELLATION_EVALUATION_BIT => { set.domain = Some(entry_point); }
                     VK_SHADER_STAGE_GEOMETRY_BIT => { set.geometry = Some(entry_point); }
-                    VK_SHADER_STAGE_FRAGMENT_BIT => { set.fragment = Some(entry_point); }
+                    VK_SHADER_STAGE_FRAGMENT_BIT if !rasterizer_discard => { set.fragment = Some(entry_point); }
```

Comment:
we also need to disable the depth/stencil states for when there is no rasterization